spell/rwa-onboarding-checklist.md

```diff
@@ -39,7 +42,7 @@
         * [ ] ensure optimization are off
         * [ ] ensure license is specified
         * [ ] ensure source matches github code and it's consistent with previous RWA onboarding
-      * [ ] constructor arg matches [chainlog](https://chainlog.makerdao.com/)
+      * [ ] constructor arg matches [changelog](https://chainlog.makerdao.com/)
```

Comment:
Nit: [changelog.makerdao.com](https://changelog.makerdao.com/) is the previous version of the contracts archive, stored on a regular server and on ipfs that was deprecated a year ago. So it's confusing to mix the terms and imply that it has to match deprecated service linking `chainlog.makerdao.com`
```suggestion
      * [ ] constructor arg matches [chainlog](https://chainlog.makerdao.com/)
```

Reply:
If "constructor arg matches chainlog" means "constructor arg matches official ChainLog address", then it's correct, since `ChainLog` is the [name of the deployed contract](https://goerli.etherscan.io/address/0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F#code). Whatever it matches `getAddress("CHANGELOG")` or not it doesn't matter, since if it's unofficially deployed chainlog, `getAddress("CHANGELOG")` can return misleading information